### DISCUSSION TITLE
Not allowed to load local resource

### DISCUSSION THREAD

**Discussion by maxigit** [qutebrowser/qutebrowser#7960] [General]
I use hoogle which start a local server with links to local files (`file://...`). When I try to follow such a link nothing happend (I get `Not allowed to load local resource` error message on the console).
Strangely, opening in another tab (right click, or `F` works).
Is there any settings to allow to load local resource ? (I tried `set content.local_content_can_access_life` but it doesn't work).

**Comment by The-Compiler** [comment]
Not sure what you mean with `content.local_content_can_access_life` exactly. If you mean `content.local_content_can_access_file_urls`, that should work. Could you maybe try to reproduce this (ideally with `--temp-basedir`) and then use `:report` so I can take a look at the logs?

**Comment by maxigit** [comment]
Sorry for the delay, I was on holiday without access to my computer
I meant `content.local_content_can_access_file_urls` indeed ( :-*)
I sent a report, not sure it is usefull.
The log in the console shows


```
Not allowed to load local resource: file:///nix/store/zkwwhvnfggwdx2dyc9r0y08zrd9jszs5-ghc-9.2.8-doc/share/doc/ghc/html/libraries/base-4.16.4.0/Data-List.html#v:map
127.0.0.1/:1 Not allowed to load local resource: file:///nix/store/zkwwhvnfggwdx2dyc9r0y08zrd9jszs5-ghc-9.2.8-doc/share/doc/ghc/html/libraries/base-4.16.4.0/Data-List.html#v:map
```

Hope this helps.

**Reply by The-Compiler** [reply]
Sorry for the delay as well, lots going on recently :sweat_smile:  Note to myself, report is `169883190712.21833.5475637547238855644@crashes.qutebrowser.org`.

Is there any difference between you clicking the link with the mouse, and using `f` to open it via hints (in the same tab)?

**Comment by maxigit** [comment]
f (in the same tab) doesn't work but displays an error message at the
bottom of the page (the same as the one displayed in the console about JS
not allowed to open local resources).
However F (hint to a new tab) works.
As well as right click : open in a new tab (which seems normal as it
bypasses JS).

**Reply by The-Compiler** [reply]
But to clarify, even a normal click reproduces the issue?

**Comment by maxigit** [comment]
Yes, normal click doesn't work (so reproduce the issue).

**Comment by maxigit** [comment]
What I understand is that there is a setting in the web engine which
forbids JS to load a local resource, which is different from the setting
set
by content.local_content_can_access_file_urls. In that case the page trying
to access the local file is not "locally" loaded but from a local server
(127.0.0.1).
